src/integration-test/integration/MappingsClient.test.ts

```diff
@@ -0,0 +1,485 @@
+
+  const mappingIds: Array<string> = [];
+  let groupId: string;
+  let groupPropertyId: string;
+  let calculatedPropertyId: string;
+  let customCalculationId: string;
+
+  let authorization: AuthorizationCallback;
+  let testIModelGroup: TestIModelGroup;
+  let testIModel: IModelMetadata;
+
+  before( async function () {
+    this.timeout(0);
+
+    const container = getTestDIContainer();
+    
+    const authorizationProvider = container.get(TestAuthorizationProvider);
+    authorization = authorizationProvider.getAdmin1Authorization();
+    accessToken = "Bearer " + (await authorization()).token;
+
+    const testIModelGroupFactory = container.get(TestIModelGroupFactory);
+    testIModelGroup = testIModelGroupFactory.create({ testRunId: getTestRunId(), packageName: TestConstants.PackagePrefix, testSuiteName: "ManagementNamedVersionOperations" });
+
+    const testIModelCreator = container.get(TestIModelCreator);
+    testIModel = await testIModelCreator.createEmptyAndUploadChangesets(testIModelGroup.getPrefixedUniqueIModelName("Test iModel for write"));
+
+    const reusableTestIModelProvider = container.get(ReusableTestIModelProvider);
+    testIModel = await reusableTestIModelProvider.getOrCreate();
+
+    //create mappings
+
+    const newMapping: MappingCreate = {
+      mappingName: "Test1"
+    };
+    let mapping = await mappingsClient.createMapping(accessToken, testIModel.id, newMapping);
+    expect(mapping).to.not.be.undefined;
+    mappingIds.push(mapping.id);
+
+    newMapping.mappingName = "Test2";
+    mapping = await mappingsClient.createMapping(accessToken, testIModel.id, newMapping);
+    expect(mapping).to.not.be.undefined;
+    mappingIds.push(mapping.id);
+
+    newMapping.mappingName = "Test3"
+    mapping = await mappingsClient.createMapping(accessToken, testIModel.id, newMapping);
+    expect(mapping).to.not.be.undefined;
+    mappingIds.push(mapping.id);
+
+    //create groups
+
+    const newGroup: GroupCreate = {
+      groupName: "Test1",
+      query: "select * from biscore.element"
+    }
+    let group = await mappingsClient.createGroup(accessToken, testIModel.id, mappingIds[0], newGroup);
+    expect(group).to.not.be.undefined;
+    groupId = group.id;
+
+    newGroup.groupName = "Test2";
+    group = await mappingsClient.createGroup(accessToken, testIModel.id, mappingIds[0], newGroup);
+    expect(group).to.not.be.undefined;
+
+    newGroup.groupName = "Test3";
+    group = await mappingsClient.createGroup(accessToken, testIModel.id, mappingIds[0], newGroup);
+    expect(group).to.not.be.undefined;
+
+    //create group properties
+
+    const ecProperty: ECProperty = {
+      ecClassName: "class",
+      ecPropertyName: "property",
+      ecPropertyType: DataType.String,
+      ecSchemaName: "schema",
+    };
+    const newProperty: GroupPropertyCreate = {
+      propertyName: "prop1",
+      dataType: DataType.Number,
+      quantityType: QuantityType.Distance,
+      ecProperties: new Array(ecProperty),
```

Comment:
Don't use `Array` constructor. Change to `[ecProperty]`.

Reply:
removed all array constructors